app/src/parser_impl.c

```diff
@@ -282,6 +290,9 @@ parser_error_t parse_additional_typebytes(parser_context_t *ctx, uint8_t type, u
         }
 
         default : {
+            // we support now generic arguments
+            // in transactions but we only support
+            // the types define above
             return parser_unexpected_type;
```

Comment:
Is there a reason we don't support them at the moment? Since the test vectors suggest they want them supported? Since we already check the type restrictions in get_type before calling this, couldn't the default here lead to "is_generic_tx"?